source/administration/upgrading-to-3.0.rst

```diff
@@ -0,0 +1,113 @@
+Upgrading to version 3.0
+========================
+
+Use these instructions if you are upgrading from 2.0.0, 2.1.0, or 2.2.0. You must upgrade to 3.0 before upgrading to the latest version of Mattermost.
```

Comment:
We seem to be inconsistent in there docs when revering to verison numbers. Sometimes it;s vX.X, other times it's X.X and other times it's version X.X. We need to pick one and be consistent